python_modules/dagster/dagster/_daemon/controller.py

```diff
@@ -75,6 +75,7 @@ def create_daemon_grpc_server_registry(
         instance_ref=instance.get_ref(),
         heartbeat_ttl=DAEMON_GRPC_SERVER_HEARTBEAT_TTL,
         startup_timeout=instance.code_server_process_startup_timeout,
+        additional_timeout_msg="To increase the timeout, set the `code_servers.local_startup_timeout` instance configuration option. ",
```

Comment:
i wonder if it will be clear what an "instance configuration option" is? you could say something like "add the following to a dagster.yaml file in your $DAGSTER_HOME folder or the folder where you are running `dagster dev`:

code_servers:
  local_startup_timeout: <timeout value>

Reply:
Good idea. Updated:

```python
/Users/ben/repos/dagster/python_modules/dagster/dagster/_core/workspace/context.py:729: UserWarning: Error loading repository location badd.py:Exception: Timed out waiting for gRPC server to start after 1s. To increase the timeout, add the following to a dagster.yaml file, located in your $DAGSTER_HOME folder or the folder where you are running `dagster dev`:

code_servers:
  local_startup_timeout: <timeout value>```